rules-emerging-threats/2025/Exploits/CVE-2025-61882/web_cve_2025_61882_oracle_e_business_exploit.yml

```diff
@@ -0,0 +1,45 @@
+title: Oracle EBS CVE-2025-61882 Exploitation Attempt
+id: f4803185-1792-4a71-86be-5e0b8f8b782d
+status: experimental
+description: |
+  Detects potential exploitation attempts targeting Oracle E-Business Suite CVE-2025-61882 vulnerability in BI Publisher Integration
+references:
+    - https://www.oracle.com/security-alerts/alert-cve-2025-61882.html
+    - https://labs.watchtowr.com/well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882/
+author: Saiprashanth Pulisetti (@0xPrashanthSec)
+date: 2025-10-08
+tags:
+    - attack.initial-access
+    - attack.execution
+    - attack.t1190
+    - attack.t1203
+    - cve.2025-61882
+    - detection.emerging-threats
+logsource:
+    category: webserver
+detection:
+    selection_paths:
+        cs-uri-stem|contains:
+            - '/OA_HTML/configurator/UiServlet'
+            - '/OA_HTML/RF.jsp'
+            - '/OA_HTML/OA.jsp'
+            - '/OA_HTML/SyncServlet'
+            - '/xmlpserver'
+            - '/OA_HTML/help/../ieshostedsurvey.jsp'
+    selection_methods:
+        cs-method:
+            - 'POST'
+            - 'GET'
+    selection_payloads:
+        cs-uri-query|contains:
+            - '<?xml'
+            - '<xsl'
+            - 'template='
+            - 'stylesheet='
+            - '../'
+            - '%2e%2e%2f'
+    condition: selection_paths and selection_methods and selection_payloads
+falsepositives:
+    - Legitimate BI Publisher report generation
+    - Administrative template uploads by authorized users
+level: critical
```

Comment:
The indicators are not focusing on the content of the requests hence the rule cannot be critical. Imo its a medium